src/components/cards/OverlayCard.tsx

```diff
@@ -17,10 +17,12 @@ const tdStyle = (backgroundColor: string): TdCSS => {
     };
 };
 
-const headlineCellStyle = {
-    width: "93%",
-    backgroundColor: palette.neutral[7],
-    padding: "3px 40px 20px 10px"
+const headlineCellStyle = (isLive: boolean): TdCSS => {
+    return {
+        width: "93%",
+        backgroundColor: isLive ? palette.news.main : palette.neutral[7],
```

Comment:
Shall we make the pillar colour dynamic here? Consider passing the pillar name into this function as an argument and then getting the pillar colour from `pillarProps`. Line 23 would look something like:
`backgroundColor: isLive ? pillarProps[pillar].colour : palette.neutral[7],`